p2p/trust/store.go

```diff
@@ -175,15 +199,19 @@ func (tms *TrustMetricStore) loadFromDB() bool {
 func (tms *TrustMetricStore) saveToDB() {
 	tms.Logger.Debug("Saving TrustHistory to DB", "size", tms.size())
 
-	peers := make(map[string]MetricHistoryJSON)
+	history := make(map[string]map[string]MetricHistoryJSON)
 
-	for key, tm := range tms.peerMetrics {
-		// Add an entry for the peer identified by key
-		peers[key] = tm.HistoryJSON()
+	for reactor, peers := range tms.reactorPeerMetrics {
```

Comment:
`peerMetrics`, not `peers`... peers is confusing

Reply:
I'll make it more clear.